recipes/large-image-converter/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
-  skip: True  # [win]
+  skip: True  # [win, py >= 36]
```

Comment:
Shouldn't this be `[win, py < 36]`, since we want to NOT skip python >= 36?

Reply:
Whoops 🤦🏻‍♂️

Reply:
So many little things going on here, its hard to stay focused